packages/ckbtc/src/bitcoin.canister.ts

```diff
@@ -42,4 +48,27 @@ export class BitcoinCanister extends Canister<BitcoinService> {
     const fn = certified ? bitcoin_get_utxos : bitcoin_get_utxos_query;
     return fn(toGetUtxosParams(params));
   };
+
+  /**
+   * Given a `get_balance_request`, which must specify a Bitcoin address and a Bitcoin network (`mainnet` or `testnet`), the function returns the current balance of this address in `Satoshi` (10^8 Satoshi = 1 Bitcoin) in the specified Bitcoin network.
+   *
+   * @link https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-bitcoin_get_utxos
```

Comment:
This link point to get_utxos. Please review the documentation if it was copied from get utxos to adapt for get balance.

Reply:
yes! sorry, forgot to update that